setup.py

```diff
@@ -10,10 +10,19 @@
         "hypothesis>=6.2.0,<7.0",  # Strategy-based fuzzer
```

Comment:
The type ignore at the top of this file should no longer be needed now that we have `types-setuptools`.